src/frontend/src/components/common/Pagination/rowsPerPage.tsx

```diff
@@ -0,0 +1,51 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+
+export interface rowsPerPageProps {
```

Comment:
interface first letter must be Upper case, other wise you can not build successful